Makefile

```diff
@@ -31,12 +31,13 @@ package-%: clean build-%
 	@echo "[ package-$* ]: Packaging exporter..."
 	bash ./scripts/package.sh $(PWD) $* $(GOOS)
 
+ACCOUNT_REGION := Staging
```

Comment:
Should we move to this region or should we clarify first? (I'm pointing it out just in case this was changed to make the tests work and we need to wait for confirmation)

Reply:
This is used locally only, since the e2e are run with the action. I set Staging here since it should be the account that we should use for local testing/dev.

Reply:
May bad! 😅 Thanks for clarifying!